tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/NativeDesktop/NativeDesktopMemoryAndStorage/Supplier Edit Native Desktop MemoryAndStorage Validation.3620.feature

```diff
@@ -0,0 +1,72 @@
+Feature:  Display Marketing Page Form Native Desktop Memory, Storage, Processing and Resolution  Validation on Edit
+    As a Supplier
+    I want to manage Marketing Page Information for the Solution's Native Desktop Connectivity Details
+    So that I can ensure the information is correct & valid
+
+Background:
+    Given Organisations exist
+        | Name     |
+        | GPs-R-Us |
+    And Suppliers exist
+        | Id    | OrganisationName |
+        | Sup 1 | GPs-R-Us         |
+    And Solutions exist
+        | SolutionID | SolutionName   | OrganisationName | SupplierStatusId | SupplierId |
+        | Sln1       | MedicOnline    | GPs-R-Us         | 1                | Sup 1      |
+
+@3620
+Scenario Outline: 1. Minimum Memory Requirement Field Fails Validation
+    Given SolutionDetail exist
+        | Solution | SummaryDescription             | FullDescription   | ClientApplication                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
For validation feature files, it's often nicer to have the client application empty, or then only contain your changes (Memory & Storage) to minimise it